web/themes/custom/novel/templates/fields/field--eventinstance--branch--full.html.twig

```diff
@@ -0,0 +1,3 @@
+{% for item in items %}
+    {{ item.content }}
+{% endfor %}
```

Comment:
Consider refactoring this.

This is [the default field template](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/e1edb7e35beacb93e78eef0541c8ccebad68d6f1/web/themes/custom/novel/templates/fields/field.html.twig#L1-L1).

I understand that we need to do something because [the default template for branches on event templates overrides this](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/e1edb7e35beacb93e78eef0541c8ccebad68d6f1/web/themes/custom/novel/templates/fields/field--eventinstance--branch.html.twig#L1-L1) but I suggest that we make the current branch field template more specific instead of double override we are doing now.